breadth_first_search/bfs.py

```diff
@@ -0,0 +1,43 @@
+edge={}
+
+def addEdges(source, destination):
+    if(source not in edge):
+        edge[source]=[]
+    edge[source].append(destination)
+
+def breadthFirstSearch(source, destination):
+    bfs=[source]
+    queue=[source]
+    visited=[source]
+    flag=0
+    

```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
+++ b/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
@@ -10,7 +10,7 @@
     queue=[source]
     visited=[source]
     flag=0
-    
+
     while(len(queue)!=0):
         source=queue.pop(0)
         temp=[]
```